build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,8 +7,8 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.1.0-preview1-24508-06";
-        public static readonly string JitVersion = "1.1.0-preview1-24508-06";
+        public static readonly string CoreCLRVersion = "1.1.0-preview1-24512-01";
+        public static readonly string JitVersion = "1.1.0-preview1-24512-01";
         public static readonly string CoreCLRVersion_Latest = "1.1.0-preview1-24508-06";
         public static readonly string JitVersion_Latest = "1.1.0-preview1-24508-06";
```

Comment:
I think @ianhays will have a fix go into master and release/1.1.0 branches today.